iOS/Sample/Podfile

```diff
@@ -1,53 +1,25 @@
 project 'Sample.xcodeproj'
 source 'https://github.com/facebook/Sonar.git'
-source 'https://github.com/CocoaPods/Specs'
+source 'https://cdn.cocoapods.org'
```

Comment:
Way faster

Reply:
Unfortunately, I will have to remove this, as our internal test is facing an issue which is mentioned [here](https://github.com/CocoaPods/CocoaPods/issues/9270) and [here](https://github.com/CocoaPods/CocoaPods/issues/9731)